test/extensions/filters/http/ext_proc/filter_test.cc

```diff
@@ -2007,15 +2011,104 @@ TEST_F(HttpFilterTest, ClearRouteCache) {
 
   EXPECT_CALL(encoder_callbacks_.downstream_callbacks_, clearRouteCache());
   processResponseBody([](const HttpBody&, ProcessingResponse&, BodyResponse& resp) {
+    auto* resp_headers_mut = resp.mutable_response()->mutable_header_mutation();
+    auto* resp_add = resp_headers_mut->add_set_headers();
+    resp_add->mutable_header()->set_key("x-new-header");
+    resp_add->mutable_header()->set_value("new");
     resp.mutable_response()->set_clear_route_cache(true);
   });
 
   filter_->onDestroy();
 
-  EXPECT_EQ(1, config_->stats().streams_started_.value());
-  EXPECT_EQ(3, config_->stats().stream_msgs_sent_.value());
-  EXPECT_EQ(3, config_->stats().stream_msgs_received_.value());
-  EXPECT_EQ(1, config_->stats().streams_closed_.value());
+  EXPECT_EQ(config_->stats().streams_started_.value(), 1);
+  EXPECT_EQ(config_->stats().stream_msgs_sent_.value(), 3);
+  EXPECT_EQ(config_->stats().stream_msgs_received_.value(), 3);
+  EXPECT_EQ(config_->stats().streams_closed_.value(), 1);
```

Comment:
didn't get why? for some code format unification?

Reply:
The ordering of the checks was modified to be clearer to read.
https://en.wikipedia.org/wiki/Yoda_conditions

Reply:
There is a c++ readability advice called `Avoid "Yoda-style" comparisons`  at Google. I didn't find its open-sourced version like other abseil tips of week. I just copied&pasted here for reference:
```
In English, which of the following sounds more natural:

"If var is equal to 42"
"If 42 is equal to var"
Likewise, in code, prefer var == 42 to 42 == var. More generally, put the "most variable" operand on the left 
(so, for example, when comparing a function local variable to a class member variable, one would usually put the local variable on the left).
```

It's not one of Envoy's [deviation](https://github.com/envoyproxy/envoy/blob/main/STYLE.md#deviations-from-google-c-style-guidelines) and I think it makes sense for new code/file follow this rule as it is more natural